Content.Client/Audio/Jukebox/JukeboxMenu.xaml.cs

```diff
@@ -90,7 +90,10 @@ public void Populate(IEnumerable<JukeboxPrototype> jukeboxProtos)
 
         foreach (var entry in jukeboxProtos)
         {
-            MusicList.AddItem(entry.Name, metadata: entry.ID);
+            //imp edit -- supporting the emag songs
+            var songName = entry.Name;
+            if(entry.EmagOnly) songName = "!#D2NT_BEATZ#!> " + entry.Name; // we literally cannot style these at time of writing, so... signify the emag songs only way we can
```

Comment:
space after `if` and put the rest of the code on a newline too. if you want to be really cool you could make `var songName` into a ternary but its no biggie